### ISSUE TITLE
Feature Request: Shortcode for User's RSVP

### ISSUE THREAD

**Issue by rachelexodus** [barryhughes/event-rocket#24]
I'd love a shortcode to display events that a user has RSVP'd to. And with that, have an event RSVP added to user data so it doesn't have to loop through all events. 

Some attributes that would be nice:
* Show Past Events RSVP'd to (bool)
* Show Past Events Up To a Certain Time (default to last 3 months maybe?)
* Show Future Events RSVP'd to (bool)
* Show Future Events Up To a Certain Time (with a default as above)
* Other attributes for whether or not to show event title, image, date, venue, etc.

Thanks for an awesome plugin!

**Comment by rachelexodus** [comment]
Now that I think about it, an event count for future and past event lists may work better than showing them up to a certain date. Or both. But I'm not asking too much am I? ;)

**Comment by barryhughes** [comment]
This should become generally available when 3.0 is released. Example usage:

*Get attendances/non-attendances for the current user (upcoming events only)*

``` html
<!-- Attendances/non-attendances for the current user -->
[eventrocket_user_rsvps]
```

*Get attendances/non-attendances for the current user (include past events)*

``` html
<!-- Past and present attendances/non-attendances for the current user -->
[eventrocket_user_rsvps only_upcoming="no"]
```

*Get attendances/non-attendances for a specific user*

``` html
<!-- To specify a user, a valid user ID must be passed -->
[eventrocket_user_rsvps user="512"]
```

*Interested only in confirmations?*

``` html
<!-- List confirmed attendances for the current user (upcoming events only) -->
[eventrocket_user_rsvps declines="no"]
```

*Interested only in confirmed absences?*

``` html
<!-- List confirmed absences for the current user (upcoming events only) -->
[eventrocket_user_rsvps attendances="no"]
```

**Comment by rachelexodus** [comment]
So excited, thank you!
